component/main.jsonnet

```
@@ -5,6 +5,31 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.airlock_microgateway;
 
+local license_secret = kube.Secret('airlock-microgateway-license') {
+  metadata+: {
+    namespace: params.namespace,
+  },
+  data_: {
+    'microgateway-license.txt': params.license,
+  },
```

What's the thinking of using `data_` here instead of just `stringData` if we put the license into the tenant repo in plain text anyway?

```suggestion
  stringData: {
    'microgateway-license.txt': params.license,
  },
```